ui/admin/app/components/workflow/steps/AddStep.tsx

```diff
@@ -93,20 +83,15 @@ export function AddStepButton({ onAddStep, className }: AddStepButtonProps) {
             <Popover open={open} onOpenChange={setOpen}>
                 <PopoverTrigger asChild>
                     <Button variant="ghost" className={className}>
-                        <Plus className="w-4 h-4 mr-2" /> Add
+                        <Plus /> Add Step
                     </Button>
                 </PopoverTrigger>
+
                 <PopoverContent
                     className="w-22 bg-secondary dark:bg-zinc-800 shadow-2xl"
                     side="top"
                 >
                     <div className="grid gap-4">
-                        {/* <Button
```

Comment:
Will this be re-enabled at a later point? Just wanted to point out commented code in case it can be deleted!